### DISCUSSION TITLE
postgresql93-server deleted

### DISCUSSION THREAD

**Discussion by arnon001** [Retrospring/retrospring#1264] [Q&A]
### Steps to reproduce the problem

I'm in the installation process,
Did portmaster databases/postgresql93-server and got this in the output:
```
root@instance-20230703-1446:~ # portmaster databases/postgresql93-server

        ===>>> The databases/postgresql93-server port moved to databases/postgresql94-server
        ===>>> Reason: Has expired: PostgreSQL-9.3 has reached end-of-life


        ===>>> The databases/postgresql94-server port moved to databases/postgresql95-server
        ===>>> Reason: Has expired: PostgreSQL-9.4 has reached end-of-life

make: chdir /usr/ports/databases/postgresql95-server: No such file or directory

===>>> The databases/postgresql95-server port has been deleted: Has expired: PostgreSQL-9.5 has reached end-of-life
===>>> Aborting update
```

### Expected behaviour

Works regular

### Actual behaviour

databases/postgresql95-server got deleted

### Specifications

Retrospring: Latest
Portmaster: Version 3.22

**Comment by nilsding** [comment]
Hi, the docs for installing it on FreeBSD are outdated (we migrated to Linux hosts a while back.)

I think running it should still work as we do not (knowingly) use anything Linux-specific.  Make sure to use a more recent version of everything though, we are currently using Postgres 14.
